dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops

```diff
@@ -296,10 +296,18 @@ INVALID_KRB5_PRINCIPAL=CWIML4558E: The {0} Kerberos principal name is incorrectl
 INVALID_KRB5_PRINCIPAL.explanation=The Kerberos principal name cannot be null. The principal name must either include the realm name or a default realm name must be defined in Kerberos configuration file.
 INVALID_KRB5_PRINCIPAL.useraction=Correct the principal name or add a default realm name.
 
-CANNOT_READ_KRB5_FILE=CWIML4559E:The {0} LdapRegistry cannot read the Kerberos file at {1}.
+CANNOT_READ_KRB5_FILE=CWIML4559E: The {0} LdapRegistry cannot read the Kerberos file at {1}.
 CANNOT_READ_KRB5_FILE.explanation=The Kerberos file cannot be opened. Either the file permissions are incorrect, or the file does not exist.
 CANNOT_READ_KRB5_FILE.useraction=Verify that the file location is correct and that the server has read file permissions.
 
 KRB5_PRINCIPAL_REQUIRED=CWIML4559E: The bind authentication mechanism on the {0} LdapRegistry is set to GSSAPI, but the required attribute, krb5Principal, is not configured.
 KRB5_PRINCIPAL_REQUIRED.explanation=The Kerberos principal name is required when GSSAPI (Kerberos) is enabled.
-KRB5_PRINCIPAL_REQUIRED.useraction=Add a krb5Principal attribute to the krbAuthentication elemenent on the LdapRegistry element.  
\ No newline at end of file
+KRB5_PRINCIPAL_REQUIRED.useraction=Add a krb5Principal attribute to the krbAuthentication elemenent on the LdapRegistry element.
+
+KRB5_FILE_NOT_FOUND=CWIML4560E: The [{0}] attribute from the {1} element is configured to a file that does not exist at: {2}
```

Comment:
is there a reason that [{0}] is double-bracketed, while the other variables are not?

is {2} a filename/filepath or a directory where the config is looking for an unnamed file that doesn't exist?

Reply:
Only that I copied these message from another message file to reuse. I removed the extra `[ ]`

`is {2} a filename/filepath or a directory where the config is looking for an unnamed file that doesn't exist?`  Yes -- here's an example. Happy to reword if desired.

`CWIML4560E: The krbTicketCache attribute from the LdapRegistry element is configured to a file that does not exist at: C:/badDir/badFile.txt`

Reply:
thanks- here's my suggestion:

The {0} attribute from the {1} element is configured to a file that does not exist at the following location: {2}